scripts/benchmark_multithreaded.py

```diff
@@ -1,18 +1,17 @@
 #!/usr/bin/env python3
 
-# To prevent _tkinter.TclError: https://stackoverflow.com/a/37605654
-import matplotlib
-matplotlib.use('Agg')
-import matplotlib.pyplot as plt
 
-import os
 import argparse
+from datetime import datetime
+import json
+import matplotlib
 import multiprocessing
+import os
 import subprocess
```

Comment:
> Do we want sorted imports?

I don't care. If you ask me, this whole `from ... import ... as ...` makes for a blood mess anyway.